meta/documents/user_guide/de/initial_setup.md

```diff
@@ -0,0 +1,187 @@
+    - Die Einstellung `Zugang` des Plentymarkets-Benutzers muss `Admin` sein
+    - Der Plentymarkets-Benutzer muss in der Lage sein, Plugin-Sets zu ändern
+
+* Aktiver Wayfair-Lieferantenstatus
+    * Eine Wayfair-Lieferanten-ID ist erforderlich
+    * [Informationen für potenzielle Lieferanten](https://partners.wayfair.com/d/onboarding/sell-on-wayfair)
+
+* [Wayfair API-Anmeldeinformationen](obtaining_credentials.md).
+
+* [Installation des Wayfair-Plugins](plugin_installation.md).
+
+
+## 1. Autorisierung des Wayfair-Plugins für den Zugriff auf Wayfair-Schnittstellen
+Nachdem das Plugin in Ihrem Plentymarkets Plugin-Set installiert wurde, muss das Plugin so konfiguriert werden, dass beim Herstellen einer Verbindung zu den Wayfair-Schnittstellen die richtigen Anmeldeinformationen verwendet werden.
+
+* **Das Autorisierungsverfahren muss für jedes Plugin-Set durchgeführt werden, welches das Wayfair-Plugin enthält.**
+* Durch Kopieren eines Plugin-Sets werden die Autorisierungsinformationen in das neue Plugin-Set kopiert.
+* Ein exportiertes oder importiertes Plugin-Set kann die Autorisierungsinformationen enthalten.
+
+Die Schritte Autorisierung sind wie folgt:
+1. Gehen Sie auf der Hauptseite von Plentymarkets zu `Plugins` >>` Plugin-Set-Übersicht`.
+
+    ![plugins_menu_plugin_set_overview](../../../images/de/plugins_menu_plugin_set_overview.png)
+
+2. Suchen Sie das Plugin-Set, welches mit dem Client verknüpft ist, mit dem Wayfair verwendet wird.
+
+    ![linked clients](../../../images/de/plugin_sets_linked_clients.png)
+
+3. Klicken Sie auf die Schaltfläche `bearbeiten` ![pencil button](../../../images/common/button_pencil.png) für das gewünschte Plugin-Set.
+
+4. Klicken Sie in der Wayfair-Zeile des Plugin-Sets auf die Schaltfläche `Einstellungen`. ![gear button](../../../images/common/button_gear.png).
+
+4. Gehen Sie im Menü auf der linken Seite zu `Konfiguration` >> `Globale Einstellungen`.
+
+    ![global settings in menu](../../../images/de/menu_global_settings.png)
+
+5. Geben Sie im Bereich `Lieferanteneinstellungen` die Werte `Client ID` und `Client Secret` ein, die Ihren Wayfair API-Anmeldeinformationen entsprechen.
+
+6. Ändern Sie die Einstellung `Modus` in "Live" - ​​siehe [Informationen zum Modus `Test`](test_mode.md)
+
+    ![global settings live](../../../images/de/global_settings_live.png)
+
+7. Klicken Sie in der Symbolleiste über den Einstellungen auf die Schaltfläche `Speichern`. ![save button](../../../images/common/button_save.png)
+
+## 2. Aktivieren der Auftragsherkunft
+Die Auftragsherkunft in Plentymarkets identifiziert den Vertriebskanal, in dem ein Auftrag generiert wurde. Damit das Plentymarkets-System Aufträge ordnungsgemäß von der Wayfair-API importieren kann, muss die Wayfair-Auftragsherkunft aktiviert sein:
+
+1. Gehen Sie auf der Hauptseite von Plentymarkets zu `Einrichtung` >>` Aufträge` >> `Auftragsherkunft`.
+
+    ![order referrer menu](../../../images/de/menu_order_referrer.png)

```

Comment:
One concern with the screenshot is that it shows other retailers/competitors with Wayfair on it that I would censor/blur/remove similar to how Jeff's screenshot has removed it

Reply:
I am not sure how Jeff removed the other plugins, but I don't think it matters if the supplier sees them

Reply:
I'll clean up the image, if it still needs it.

Reply:
image cleaned up.